spec/javascripts/global-bar-class-toggle.spec.js

```diff
@@ -0,0 +1,119 @@
+describe("toggling a global bar HTML class based on cookie", function () {
+  var root = window;
+
+  function globalBarSource(fakeWindow) {
+    var window = fakeWindow || root;
+
+    /* --------------------------------------- */
```

Comment:
What are these dividing lines for? They're pretty ugly, and I don't think we have anything like this in our current codebase.

Reply:
Clarification added here https://github.com/alphagov/static/pull/1583/commits/39ce98444f066724a5a44d9179dd2a625fcf9dc0